doc/concepts/cell.rst

```diff
@@ -14,10 +14,10 @@ Cells interact with each other via spike exchange and gap junctions.
     =============================  ===========================================  ===========================================================
     .. generic:: gid               integral                                     The unique global identifier of a cell.
     .. generic:: tag               string                                       The label of a group of items in a cell-local collection.
-                                                                                For example the synapses "syns_0" on a cell.
+                                                                                For example, the synapse "syns_0" on a cell.
```

Comment:
A label might refer to more than a single synapse (item in general). Maybe this is clearer?
```suggestion
                                                                                For example, the synapses labelled ``"syns_0"`` on a cell.
```